.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ${{ matrix.platform }}
     steps:
       - name: Install Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
```

Comment:
Are they still supporting the `.x` versions? https://github.com/actions/setup-go#supported-version-syntax

> The go-version input supports the following syntax:
>
> - Specific versions: `1.15`, `1.16.1`, `1.17.0-rc.2`, `1.16.0-beta.1`
> - SemVer's version range syntax: `^1.13.1`, `>=1.18.0-rc.1`

Reply:
Looks like they do; it's documented in https://github.com/npm/node-semver

> Any of `X`, `x`, or `*` may be used to "stand in" for one of the numeric values in the [major, minor, patch] tuple.